projects/proj-event-cards.md

```diff
@@ -0,0 +1,105 @@
+- Location of the event
+- When it is
+- Link to the website of the band, festival, or where you can buy tickets
+
+We want to build these cards with the following in mind:
+- A "Card" is a small piece of a website, not usually the whole page. We'll probably want to put multiple cards on the final page.
+- Put things in the order that makes the most sense for HTML.
+- Research semantic tags for each item; it is unlikely you'll be using `<div>` very often, but you may decide that it's the best choice some places.
+- Creating [accessible elements may require additional attributes](https://www.w3.org/WAI/standards-guidelines/wcag/).
+- Inline comments are helpful for reminding yourself why you made a decision you made.
+- Focus on semantically strong, accessible Markup. Styles will come later, and if you do them now, we'll probably completely ignore them anyway.
+
+Next week we will spend 5 minutes or so going over the HTML choices each of you have made. Try to do this individually (not as a group). Please do take advantage of office hours or pairing with developers!
+
+## Week 2 - CSS
+
+This week, you've been given a single event card to style with _vanilla_ CSS. Using the HTML you've already written (and adapted where necessary), try to get as close to the design as possible by next Thursday!
+
+I really encourage you this week to make time to pair with Frontend Designers, as well as the rest of the frontend-focused team members mentioned before. Another awesome resource is [MDN](https://developer.mozilla.org/en-US/).
+
+Next week we will spend 5 minutes or so going over the CSS choices each of you have made. Try to do this individually and not as a group.
+
+## Week 3 - Layout
+
+In addition to the individual card improvements we've asked you to do, your next task is to create a grid layout with your event cards. This layout should be [responsive](https://responsivedesign.is/), developed with a [mobile first](https://vimeo.com/38187066) frame of mind, and all content should be browser tested to the specifications listed below.
+
+If you want bonus points, find an open source svg icon library you like for later use.
+
+*Note*: For this task, we can utilize Codepen's native features to link pens together.
+
+To include the HTML from another pen, add the pen url, surrounded by three brackets.
+```html
+[[[https://<url to your event card>]]]
+```
+
+You will also need to link to your pen as an external css resource by going to *Settings -> CSS - > Add External Stylesheets/Pens* and adding the URL of your card.
+
+### Browser Testing
+*Note*: This is a big ask. It's not fun. It won't be perfect. You don't have the full resources of a design team. Do the best you can, but don't burn out.
+
+At Sparkbox, we don't have Quality Assurance, so we usually do our own browser testing. If there is interest, we can do a 'bug bash' together, to learn about the device wall and [BrowserStack](https://www.browserstack.com/start).
+
+
+#### Gold Level Support
```

Comment:
Not a hill I'll die on, but I can see this list changing at a different cadence than the rest of the curriculum. For that reason, you might want to put this in a different file with a link.